docs/standard/data/xml/xslt-transformations.md

```diff
@@ -38,13 +38,9 @@ The Extensible Stylesheet Language Transformation (XSLT) lets you transform the
  [XSLT Transformations with the XslTransform Class](../../../../docs/standard/data/xml/xslt-transformations-with-the-xsltransform-class.md)  
```

Comment:
We should file an issue to review this area. We shouldn't have topics that cover classes that are obsolete for so long. That should probably be changed to use XslCompiledTransform instead.

Reply:
Actually we un-obsoleting this type :smile: there are a few scenarios that one should use `XslTransform` see https://github.com/dotnet/corefx/issues/25874 and a brief description of why at  https://github.com/dotnet/corefx/issues/25874#issuecomment-351465098

Reply:
Thanks for the heads-up. I see that I missed the tag on that issue. Let's add an issue for the required docs work then to un-obsolete that type. 😄